### ISSUE TITLE
Zip extraction failure on external sd and missing path separator in extracted folders

### ISSUE THREAD

**Issue by ClockGen** [SimpleMobileTools/Simple-File-Manager#308]
OS: LineageOS 15.1 (Oreo)
App version: 6.1.3 (Latest from F-Droid)
It seems that #240 is back.  
Also, while trying to unpack any zip located on an external sd, I get a toast notification with the following error `An error occured: java.util.zip.ZipException: zip file is empty`

**Comment by tibbi** [comment]
retry with 6.3.0

**Comment by ClockGen** [comment]
@tibbi Yes, now it extracts folders properly on internal memory. Still getting the same error on external though.
